macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -28,7 +28,6 @@
 		33CC11132044BFA00003C045 /* MainFlutterWindow.swift in Sources */ = {isa = PBXBuildFile; fileRef = 33CC11122044BFA00003C045 /* MainFlutterWindow.swift */; };
```

Comment:
It looks like you deleted the gallery macOS app icon and replaced it with the default flutter icon. Github wouldn't let me comment on the image so I commented here. I don't think we want to do that.

Reply:
Yeah, I already reverted that. Thanks for pointing it out.